.github/workflows/create-hotfix-branch.yml

```diff
@@ -12,10 +14,44 @@ jobs:
     name: Create New Branch
     runs-on: ubuntu-latest
 
-    # Only allow release stakeholders to initiate releases
-    if: github.ref == 'refs/heads/main' && (github.actor == 'ItsSudip' || github.actor == 'krishna2020' || github.actor == 'lvrach' || github.actor == 'cisse21' || github.actor == 'debanjan97' || github.actor == 'saikumarrs' || github.actor == 'sandeepdsvs' || github.actor == 'utsabc' || github.actor == 'yashasvibajpai' || github.actor == 'maheshkutty' || github.actor == 'manish339k') && (github.triggering_actor == 'ItsSudip' || github.triggering_actor == 'krishna2020' || github.triggering_actor == 'lvrach' || github.triggering_actor == 'cisse21' || github.triggering_actor == 'debanjan97' || github.triggering_actor == 'saikumarrs' || github.triggering_actor == 'sandeepdsvs' || github.triggering_actor == 'utsabc' || github.triggering_actor == 'yashasvibajpai' || github.triggering_actor == 'maheshkutty' || github.triggering_actor == 'manish339k')
     steps:
+      - name: Check team membership or allowlist
+        id: check_access
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const allowedActors = [
+              "ItsSudip", "krishna2020", "lvrach", "cisse21", "debanjan97",
+              "saikumarrs", "sandeepdsvs", "utsabc", "yashasvibajpai",
+              "maheshkutty", "manish339k"
```

Comment:
Ideally, we should create a team for these users as well. It's hard to keep up with a dynamic list.

Reply:
this will do later, after discussing with then team on how to go about it